qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -499,22 +337,11 @@ class GroverResult(AlgorithmResult):
 
     def __init__(self) -> None:
```

Comment:
So at the moment we do not have the notion of an AmplicationResult, where if needed GroverResult would extend and add unique fields related to Grover?

Reply:
No, not yet, but we could add it without an API change it necessary